src/connection/host_lookup.c

```diff
@@ -462,6 +433,46 @@ ip_format_check (char *ip_addr)
   return ret;
 }
 
+/*
+ * hostname_format_check () - Check the host name is valid format.
+ *
+ * return   : true if host name is valid format, false otherwise
+ */
+static bool
+hostname_format_check (char *hostname, int str_len)
+{
+
+  int char_num = 0;
+
+  if (!((hostname[0] >= 'A' && hostname[0] <= 'Z') ^ (hostname[0] >= 'a' && hostname[0] <= 'z')))
```

Comment:
For the readability, I propose:

`if (!isalpha (hostname[0]))`

Reply:
It will be better. I will apply it.